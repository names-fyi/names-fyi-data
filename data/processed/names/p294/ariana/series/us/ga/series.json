[{"y":1986,"t":5,"m":0,"f":5,"mr":null,"fr":808,"mp":0,"fp":0.0001326576636332281},{"y":1987,"t":6,"m":0,"f":6,"mr":null,"fr":746,"mp":0,"fp":0.00015310418739952538},{"y":1989,"t":8,"m":0,"f":8,"mr":null,"fr":637,"mp":0,"fp":0.0001912000191200019},{"y":1990,"t":15,"m":0,"f":15,"mr":null,"fr":418,"mp":0,"fp":0.00035148561252226074},{"y":1991,"t":27,"m":0,"f":27,"mr":null,"fr":253,"mp":0,"fp":0.0006450533960866761},{"y":1992,"t":17,"m":0,"f":17,"mr":null,"fr":361,"mp":0,"fp":0.0004116920543433512},{"y":1993,"t":22,"m":0,"f":22,"mr":null,"fr":290,"mp":0,"fp":0.0005436528529419032},{"y":1994,"t":33,"m":0,"f":33,"mr":null,"fr":222,"mp":0,"fp":0.0008098358241920048},{"y":1995,"t":27,"m":0,"f":27,"mr":null,"fr":269,"mp":0,"fp":0.0006597111930999095},{"y":1996,"t":29,"m":0,"f":29,"mr":null,"fr":266,"mp":0,"fp":0.00069867733153444},{"y":1997,"t":36,"m":0,"f":36,"mr":null,"fr":234,"mp":0,"fp":0.000841947705692502},{"y":1998,"t":42,"m":0,"f":42,"mr":null,"fr":218,"mp":0,"fp":0.0009430784776018862},{"y":1999,"t":62,"m":0,"f":62,"mr":null,"fr":154,"mp":0,"fp":0.0013557839492674393},{"y":2000,"t":64,"m":0,"f":64,"mr":null,"fr":158,"mp":0,"fp":0.001331779590477776},{"y":2001,"t":80,"m":0,"f":80,"mr":null,"fr":126,"mp":0,"fp":0.001658512314453935},{"y":2002,"t":56,"m":0,"f":56,"mr":null,"fr":174,"mp":0,"fp":0.001156953081420573},{"y":2003,"t":85,"m":0,"f":85,"mr":null,"fr":126,"mp":0,"fp":0.001708886208283072},{"y":2004,"t":89,"m":0,"f":89,"mr":null,"fr":121,"mp":0,"fp":0.0017616436729280894},{"y":2005,"t":88,"m":0,"f":88,"mr":null,"fr":117,"mp":0,"fp":0.0017125620317213195},{"y":2006,"t":112,"m":0,"f":112,"mr":null,"fr":93,"mp":0,"fp":0.0020811267815002694},{"y":2007,"t":108,"m":0,"f":108,"mr":null,"fr":105,"mp":0,"fp":0.0019923258559622196},{"y":2008,"t":87,"m":0,"f":87,"mr":null,"fr":129,"mp":0,"fp":0.0016478833222843073},{"y":2009,"t":98,"m":0,"f":98,"mr":null,"fr":109,"mp":0,"fp":0.0019327482496795188},{"y":2010,"t":81,"m":0,"f":81,"mr":null,"fr":127,"mp":0,"fp":0.0016900703152711416},{"y":2011,"t":95,"m":0,"f":95,"mr":null,"fr":103,"mp":0,"fp":0.0019922407465660062},{"y":2012,"t":96,"m":0,"f":96,"mr":null,"fr":102,"mp":0,"fp":0.00203028508586414},{"y":2013,"t":123,"m":0,"f":123,"mr":null,"fr":75,"mp":0,"fp":0.0026440809131752616},{"y":2014,"t":158,"m":0,"f":158,"mr":null,"fr":56,"mp":0,"fp":0.0032805265452733427},{"y":2015,"t":159,"m":0,"f":159,"mr":null,"fr":56,"mp":0,"fp":0.003269855632789043},{"y":2016,"t":144,"m":0,"f":144,"mr":null,"fr":56,"mp":0,"fp":0.0030010628764354044},{"y":2017,"t":126,"m":0,"f":126,"mr":null,"fr":68,"mp":0,"fp":0.0026776606596395783},{"y":2018,"t":138,"m":0,"f":138,"mr":null,"fr":54,"mp":0,"fp":0.002987594986036241},{"y":2019,"t":116,"m":0,"f":116,"mr":null,"fr":70,"mp":0,"fp":0.0025069156293222685},{"y":2020,"t":103,"m":0,"f":103,"mr":null,"fr":79,"mp":0,"fp":0.0023226446579172866},{"y":2021,"t":94,"m":0,"f":94,"mr":null,"fr":92,"mp":0,"fp":0.002082687109496167},{"y":2022,"t":103,"m":0,"f":103,"mr":null,"fr":78,"mp":0,"fp":0.002265728112626485}]