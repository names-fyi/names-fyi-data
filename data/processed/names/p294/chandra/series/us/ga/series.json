[{"y":1954,"t":5,"m":0,"f":5,"mr":null,"fr":615,"mp":0,"fp":0.00010884235273629675},{"y":1956,"t":7,"m":0,"f":7,"mr":null,"fr":548,"mp":0,"fp":0.0001498415960270571},{"y":1957,"t":8,"m":0,"f":8,"mr":null,"fr":505,"mp":0,"fp":0.00017390548237033172},{"y":1958,"t":20,"m":0,"f":20,"mr":null,"fr":316,"mp":0,"fp":0.00044327223564352046},{"y":1959,"t":20,"m":0,"f":20,"mr":null,"fr":322,"mp":0,"fp":0.0004468474909513383},{"y":1960,"t":7,"m":0,"f":7,"mr":null,"fr":568,"mp":0,"fp":0.0001571621014818141},{"y":1961,"t":19,"m":0,"f":19,"mr":null,"fr":341,"mp":0,"fp":0.0004266020027841394},{"y":1962,"t":17,"m":0,"f":17,"mr":null,"fr":345,"mp":0,"fp":0.00038929217522727796},{"y":1963,"t":22,"m":0,"f":22,"mr":null,"fr":295,"mp":0,"fp":0.0005044020542920029},{"y":1964,"t":18,"m":0,"f":18,"mr":null,"fr":347,"mp":0,"fp":0.0004102470598960707},{"y":1965,"t":22,"m":0,"f":22,"mr":null,"fr":286,"mp":0,"fp":0.0005386479935362241},{"y":1966,"t":22,"m":0,"f":22,"mr":null,"fr":280,"mp":0,"fp":0.0005687252798386889},{"y":1967,"t":30,"m":0,"f":30,"mr":null,"fr":224,"mp":0,"fp":0.0008134490238611713},{"y":1968,"t":32,"m":0,"f":32,"mr":null,"fr":215,"mp":0,"fp":0.000849933598937583},{"y":1969,"t":23,"m":0,"f":23,"mr":null,"fr":280,"mp":0,"fp":0.0006023465325790908},{"y":1970,"t":40,"m":0,"f":40,"mr":null,"fr":207,"mp":0,"fp":0.0009896580731357317},{"y":1971,"t":45,"m":0,"f":45,"mr":null,"fr":187,"mp":0,"fp":0.0011586292129045547},{"y":1972,"t":42,"m":0,"f":42,"mr":null,"fr":185,"mp":0,"fp":0.0011672178528749687},{"y":1973,"t":40,"m":0,"f":40,"mr":null,"fr":186,"mp":0,"fp":0.0011739845034045551},{"y":1974,"t":43,"m":0,"f":43,"mr":null,"fr":172,"mp":0,"fp":0.0013029908184600467},{"y":1975,"t":45,"m":0,"f":45,"mr":null,"fr":151,"mp":0,"fp":0.0014525969204945285},{"y":1976,"t":25,"m":0,"f":25,"mr":null,"fr":250,"mp":0,"fp":0.0008094020137922103},{"y":1977,"t":39,"m":0,"f":39,"mr":null,"fr":176,"mp":0,"fp":0.0011873230432002922},{"y":1978,"t":22,"m":0,"f":22,"mr":null,"fr":273,"mp":0,"fp":0.0006822128504093277},{"y":1979,"t":23,"m":0,"f":23,"mr":null,"fr":272,"mp":0,"fp":0.0006843201428146385},{"y":1980,"t":27,"m":0,"f":27,"mr":null,"fr":235,"mp":0,"fp":0.0007651759904778099},{"y":1981,"t":19,"m":0,"f":19,"mr":null,"fr":297,"mp":0,"fp":0.000545600735125201},{"y":1982,"t":21,"m":0,"f":21,"mr":null,"fr":278,"mp":0,"fp":0.0006019261637239165},{"y":1983,"t":22,"m":0,"f":22,"mr":null,"fr":265,"mp":0,"fp":0.000626655652718831},{"y":1984,"t":19,"m":0,"f":19,"mr":null,"fr":306,"mp":0,"fp":0.0005323620061641916},{"y":1985,"t":19,"m":0,"f":19,"mr":null,"fr":302,"mp":0,"fp":0.0005121017734892998},{"y":1986,"t":13,"m":0,"f":13,"mr":null,"fr":417,"mp":0,"fp":0.00034490992544639304},{"y":1987,"t":15,"m":0,"f":15,"mr":null,"fr":382,"mp":0,"fp":0.0003827604684988134},{"y":1988,"t":12,"m":0,"f":12,"mr":null,"fr":464,"mp":0,"fp":0.00029911760307094074},{"y":1989,"t":9,"m":0,"f":9,"mr":null,"fr":595,"mp":0,"fp":0.00021510002151000216},{"y":1990,"t":7,"m":0,"f":7,"mr":null,"fr":752,"mp":0,"fp":0.000164026619177055},{"y":1991,"t":8,"m":0,"f":8,"mr":null,"fr":686,"mp":0,"fp":0.00019112693217382995},{"y":1992,"t":5,"m":0,"f":5,"mr":null,"fr":1008,"mp":0,"fp":0.00012108589833627975},{"y":1993,"t":6,"m":0,"f":6,"mr":null,"fr":845,"mp":0,"fp":0.00014826895989324636},{"y":1994,"t":7,"m":0,"f":7,"mr":null,"fr":777,"mp":0,"fp":0.00017178335664678888}]