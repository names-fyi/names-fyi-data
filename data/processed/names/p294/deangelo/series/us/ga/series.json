[{"y":1972,"t":5,"m":5,"f":0,"mr":572,"fr":null,"mp":0.00012328936012822092,"fp":0},{"y":1975,"t":5,"m":5,"f":0,"mr":582,"fr":null,"mp":0.00013719679508286686,"fp":0},{"y":1976,"t":10,"m":10,"f":0,"mr":386,"fr":null,"mp":0.0002806544862619629,"fp":0},{"y":1977,"t":7,"m":7,"f":0,"mr":501,"fr":null,"mp":0.00018484776466238877,"fp":0},{"y":1978,"t":5,"m":5,"f":0,"mr":629,"fr":null,"mp":0.00013232764324467382,"fp":0},{"y":1979,"t":8,"m":8,"f":0,"mr":468,"fr":null,"mp":0.0002035727009008092,"fp":0},{"y":1980,"t":9,"m":9,"f":0,"mr":458,"fr":null,"mp":0.00021838299524410367,"fp":0},{"y":1981,"t":10,"m":10,"f":0,"mr":407,"fr":null,"mp":0.00024820670654521085,"fp":0},{"y":1982,"t":6,"m":6,"f":0,"mr":564,"fr":null,"mp":0.00014785608674223755,"fp":0},{"y":1983,"t":16,"m":16,"f":0,"mr":292,"fr":null,"mp":0.0003949252110381596,"fp":0},{"y":1984,"t":26,"m":26,"f":0,"mr":228,"fr":null,"mp":0.000623306882746386,"fp":0},{"y":1985,"t":23,"m":23,"f":0,"mr":253,"fr":null,"mp":0.0005324936911073554,"fp":0},{"y":1986,"t":25,"m":25,"f":0,"mr":248,"fr":null,"mp":0.0005657900692527045,"fp":0},{"y":1987,"t":24,"m":24,"f":0,"mr":257,"fr":null,"mp":0.0005268472581990604,"fp":0},{"y":1988,"t":30,"m":30,"f":0,"mr":239,"fr":null,"mp":0.000642370776412145,"fp":0},{"y":1989,"t":33,"m":33,"f":0,"mr":237,"fr":null,"mp":0.0006670709520921771,"fp":0},{"y":1990,"t":34,"m":34,"f":0,"mr":236,"fr":null,"mp":0.0006778579688185335,"fp":0},{"y":1991,"t":33,"m":33,"f":0,"mr":232,"fr":null,"mp":0.0006784399991776485,"fp":0},{"y":1992,"t":24,"m":24,"f":0,"mr":296,"fr":null,"mp":0.0004922370121213365,"fp":0},{"y":1993,"t":17,"m":17,"f":0,"mr":373,"fr":null,"mp":0.0003534450496902158,"fp":0},{"y":1994,"t":20,"m":20,"f":0,"mr":342,"fr":null,"mp":0.000416181122024305,"fp":0},{"y":1995,"t":20,"m":20,"f":0,"mr":357,"fr":null,"mp":0.0004131121806126454,"fp":0},{"y":1996,"t":23,"m":23,"f":0,"mr":339,"fr":null,"mp":0.000474765197646816,"fp":0},{"y":1997,"t":27,"m":27,"f":0,"mr":307,"fr":null,"mp":0.0005380415288350405,"fp":0},{"y":1998,"t":18,"m":18,"f":0,"mr":432,"fr":null,"mp":0.0003489657044260483,"fp":0},{"y":1999,"t":18,"m":18,"f":0,"mr":433,"fr":null,"mp":0.00033459114820529024,"fp":0},{"y":2000,"t":15,"m":15,"f":0,"mr":509,"fr":null,"mp":0.0002707874499043218,"fp":0},{"y":2001,"t":10,"m":10,"f":0,"mr":658,"fr":null,"mp":0.00017991112390479104,"fp":0},{"y":2002,"t":16,"m":16,"f":0,"mr":505,"fr":null,"mp":0.0002867332126664397,"fp":0},{"y":2003,"t":10,"m":10,"f":0,"mr":706,"fr":null,"mp":0.0001749965000699986,"fp":0},{"y":2004,"t":10,"m":10,"f":0,"mr":727,"fr":null,"mp":0.00017147070423018226,"fp":0},{"y":2005,"t":14,"m":14,"f":0,"mr":616,"fr":null,"mp":0.0002344783693704256,"fp":0},{"y":2006,"t":17,"m":17,"f":0,"mr":552,"fr":null,"mp":0.0002723398801704527,"fp":0},{"y":2007,"t":9,"m":9,"f":0,"mr":857,"fr":null,"mp":0.00014270990248156664,"fp":0},{"y":2008,"t":20,"m":20,"f":0,"mr":503,"fr":null,"mp":0.000326963004136082,"fp":0},{"y":2009,"t":18,"m":18,"f":0,"mr":537,"fr":null,"mp":0.00030507957492245896,"fp":0},{"y":2010,"t":17,"m":17,"f":0,"mr":531,"fr":null,"mp":0.00030433225921947725,"fp":0},{"y":2011,"t":17,"m":17,"f":0,"mr":517,"fr":null,"mp":0.0003093834170488462,"fp":0},{"y":2012,"t":15,"m":15,"f":0,"mr":565,"fr":null,"mp":0.0002779167361458507,"fp":0},{"y":2013,"t":18,"m":18,"f":0,"mr":508,"fr":null,"mp":0.00033552668369153915,"fp":0},{"y":2014,"t":10,"m":10,"f":0,"mr":776,"fr":null,"mp":0.00018194388850478513,"fp":0},{"y":2015,"t":12,"m":12,"f":0,"mr":715,"fr":null,"mp":0.00021859117984589322,"fp":0},{"y":2016,"t":6,"m":6,"f":0,"mr":1116,"fr":null,"mp":0.00011171520071497728,"fp":0},{"y":2017,"t":5,"m":5,"f":0,"mr":1284,"fr":null,"mp":0.00009325922333718805,"fp":0},{"y":2018,"t":13,"m":13,"f":0,"mr":662,"fr":null,"mp":0.00024946269573226893,"fp":0},{"y":2020,"t":5,"m":5,"f":0,"mr":1348,"fr":null,"mp":0.00009976256509507372,"fp":0},{"y":2021,"t":5,"m":5,"f":0,"mr":1345,"fr":null,"mp":0.0000995837399669382,"fp":0},{"y":2022,"t":6,"m":6,"f":0,"mr":1233,"fr":null,"mp":0.00011795930404010616,"fp":0}]