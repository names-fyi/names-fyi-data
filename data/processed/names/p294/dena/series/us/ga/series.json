[{"y":1918,"t":5,"m":0,"f":5,"mr":null,"fr":545,"mp":0,"fp":0.00014565793690098173},{"y":1932,"t":5,"m":0,"f":5,"mr":null,"fr":494,"mp":0,"fp":0.00016268098259313488},{"y":1944,"t":5,"m":0,"f":5,"mr":null,"fr":524,"mp":0,"fp":0.00014020132910859995},{"y":1950,"t":5,"m":0,"f":5,"mr":null,"fr":584,"mp":0,"fp":0.00011947145826861962},{"y":1952,"t":5,"m":0,"f":5,"mr":null,"fr":610,"mp":0,"fp":0.0001135125317835089},{"y":1953,"t":9,"m":0,"f":9,"mr":null,"fr":458,"mp":0,"fp":0.00020194314178652366},{"y":1954,"t":6,"m":0,"f":6,"mr":null,"fr":563,"mp":0,"fp":0.0001306108232835561},{"y":1955,"t":11,"m":0,"f":11,"mr":null,"fr":427,"mp":0,"fp":0.0002389330553021417},{"y":1956,"t":16,"m":0,"f":16,"mr":null,"fr":347,"mp":0,"fp":0.0003424950766332734},{"y":1957,"t":8,"m":0,"f":8,"mr":null,"fr":513,"mp":0,"fp":0.00017390548237033172},{"y":1958,"t":7,"m":0,"f":7,"mr":null,"fr":573,"mp":0,"fp":0.00015514528247523218},{"y":1959,"t":12,"m":0,"f":12,"mr":null,"fr":420,"mp":0,"fp":0.00026810849457080296},{"y":1960,"t":7,"m":0,"f":7,"mr":null,"fr":578,"mp":0,"fp":0.0001571621014818141},{"y":1961,"t":22,"m":0,"f":22,"mr":null,"fr":303,"mp":0,"fp":0.0004939602137500562},{"y":1962,"t":29,"m":0,"f":29,"mr":null,"fr":254,"mp":0,"fp":0.0006640866518582976},{"y":1963,"t":21,"m":0,"f":21,"mr":null,"fr":310,"mp":0,"fp":0.000481474688187821},{"y":1964,"t":32,"m":0,"f":32,"mr":null,"fr":241,"mp":0,"fp":0.0007293281064819036},{"y":1965,"t":21,"m":0,"f":21,"mr":null,"fr":299,"mp":0,"fp":0.000514163993830032},{"y":1966,"t":22,"m":0,"f":22,"mr":null,"fr":282,"mp":0,"fp":0.0005687252798386889},{"y":1967,"t":27,"m":0,"f":27,"mr":null,"fr":238,"mp":0,"fp":0.0007321041214750542},{"y":1968,"t":31,"m":0,"f":31,"mr":null,"fr":223,"mp":0,"fp":0.0008233731739707835},{"y":1969,"t":34,"m":0,"f":34,"mr":null,"fr":216,"mp":0,"fp":0.0008904253090299602},{"y":1970,"t":57,"m":0,"f":57,"mr":null,"fr":162,"mp":0,"fp":0.0014102627542184176},{"y":1971,"t":29,"m":0,"f":29,"mr":null,"fr":251,"mp":0,"fp":0.0007466721594273797},{"y":1972,"t":28,"m":0,"f":28,"mr":null,"fr":243,"mp":0,"fp":0.0007781452352499792},{"y":1973,"t":25,"m":0,"f":25,"mr":null,"fr":255,"mp":0,"fp":0.0007337403146278469},{"y":1974,"t":17,"m":0,"f":17,"mr":null,"fr":326,"mp":0,"fp":0.0005151359049725766},{"y":1975,"t":23,"m":0,"f":23,"mr":null,"fr":267,"mp":0,"fp":0.0007424384260305368},{"y":1976,"t":22,"m":0,"f":22,"mr":null,"fr":276,"mp":0,"fp":0.000712273772137145},{"y":1977,"t":16,"m":0,"f":16,"mr":null,"fr":352,"mp":0,"fp":0.00048710688951806864},{"y":1978,"t":20,"m":0,"f":20,"mr":null,"fr":298,"mp":0,"fp":0.0006201935003721161},{"y":1979,"t":14,"m":0,"f":14,"mr":null,"fr":390,"mp":0,"fp":0.0004165426956263017},{"y":1980,"t":13,"m":0,"f":13,"mr":null,"fr":416,"mp":0,"fp":0.00036841806948931585},{"y":1981,"t":14,"m":0,"f":14,"mr":null,"fr":376,"mp":0,"fp":0.0004020215943027797},{"y":1982,"t":11,"m":0,"f":11,"mr":null,"fr":459,"mp":0,"fp":0.0003152946571887182},{"y":1984,"t":16,"m":0,"f":16,"mr":null,"fr":348,"mp":0,"fp":0.0004483048472961614},{"y":1985,"t":5,"m":0,"f":5,"mr":null,"fr":834,"mp":0,"fp":0.0001347636246024473},{"y":1986,"t":10,"m":0,"f":10,"mr":null,"fr":509,"mp":0,"fp":0.0002653153272664562},{"y":1987,"t":5,"m":0,"f":5,"mr":null,"fr":868,"mp":0,"fp":0.0001275868228329378},{"y":1989,"t":5,"m":0,"f":5,"mr":null,"fr":953,"mp":0,"fp":0.0001195000119500012},{"y":1991,"t":5,"m":0,"f":5,"mr":null,"fr":998,"mp":0,"fp":0.00011945433260864371}]