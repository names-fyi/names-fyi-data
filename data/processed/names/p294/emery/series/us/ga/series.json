[{"y":1911,"t":5,"m":5,"f":0,"mr":247,"fr":null,"mp":0.0005587216448765225,"fp":0},{"y":1912,"t":5,"m":5,"f":0,"mr":349,"fr":null,"mp":0.0003417167851284855,"fp":0},{"y":1914,"t":9,"m":9,"f":0,"mr":286,"fr":null,"mp":0.00042851021282673904,"fp":0},{"y":1916,"t":7,"m":7,"f":0,"mr":388,"fr":null,"mp":0.0002679938744257274,"fp":0},{"y":1917,"t":9,"m":9,"f":0,"mr":323,"fr":null,"mp":0.00034374761286379957,"fp":0},{"y":1918,"t":5,"m":5,"f":0,"mr":527,"fr":null,"mp":0.00017102202763715965,"fp":0},{"y":1919,"t":7,"m":7,"f":0,"mr":430,"fr":null,"mp":0.00022568996646891927,"fp":0},{"y":1921,"t":6,"m":6,"f":0,"mr":462,"fr":null,"mp":0.0001923385157877865,"fp":0},{"y":1922,"t":5,"m":5,"f":0,"mr":528,"fr":null,"mp":0.0001625276296970485,"fp":0},{"y":1923,"t":9,"m":9,"f":0,"mr":329,"fr":null,"mp":0.00031861790632633553,"fp":0},{"y":1927,"t":5,"m":5,"f":0,"mr":473,"fr":null,"mp":0.00017548169725897588,"fp":0},{"y":1928,"t":5,"m":5,"f":0,"mr":465,"fr":null,"mp":0.00017789795773144525,"fp":0},{"y":1929,"t":7,"m":7,"f":0,"mr":360,"fr":null,"mp":0.0002599138571216397,"fp":0},{"y":1935,"t":5,"m":5,"f":0,"mr":461,"fr":null,"mp":0.00017477628635346757,"fp":0},{"y":1937,"t":5,"m":5,"f":0,"mr":463,"fr":null,"mp":0.00017179178835251675,"fp":0},{"y":1946,"t":5,"m":5,"f":0,"mr":497,"fr":null,"mp":0.00012419274714356682,"fp":0},{"y":1951,"t":5,"m":5,"f":0,"mr":514,"fr":null,"mp":0.00011132138483802739,"fp":0},{"y":1953,"t":5,"m":5,"f":0,"mr":539,"fr":null,"mp":0.00010653257766224912,"fp":0},{"y":1954,"t":5,"m":5,"f":0,"mr":531,"fr":null,"mp":0.00010305028854080792,"fp":0},{"y":1960,"t":6,"m":6,"f":0,"mr":506,"fr":null,"mp":0.0001244090569793481,"fp":0},{"y":1961,"t":5,"m":5,"f":0,"mr":582,"fr":null,"mp":0.00010459156991946449,"fp":0},{"y":1996,"t":5,"m":5,"f":0,"mr":916,"fr":null,"mp":0.00010320982557539478,"fp":0},{"y":1998,"t":5,"m":0,"f":5,"mr":null,"fr":1202,"mp":0,"fp":0.00011227124733355787},{"y":2002,"t":5,"m":0,"f":5,"mr":null,"fr":1399,"mp":0,"fp":0.00010329938226969402},{"y":2003,"t":7,"m":0,"f":7,"mr":null,"fr":1117,"mp":0,"fp":0.0001407318053880177},{"y":2004,"t":12,"m":0,"f":12,"mr":null,"fr":752,"mp":0,"fp":0.0002375249896082817},{"y":2005,"t":26,"m":13,"f":13,"mr":652,"fr":749,"mp":0.00021772991441539518,"fp":0.00025299211832246766},{"y":2006,"t":17,"m":8,"f":9,"mr":927,"fr":1027,"mp":0.00012815994360962482,"fp":0.00016723340208484307},{"y":2007,"t":15,"m":0,"f":15,"mr":null,"fr":707,"mp":0,"fp":0.0002767119244391972},{"y":2008,"t":24,"m":6,"f":18,"mr":1166,"fr":593,"mp":0.0000980889012408246,"fp":0.0003409413770243394},{"y":2009,"t":49,"m":11,"f":38,"mr":764,"fr":287,"mp":0.00018643751800816936,"fp":0.000749432994773691},{"y":2010,"t":31,"m":9,"f":22,"mr":848,"fr":479,"mp":0.00016111707841031148,"fp":0.0004590314436538903},{"y":2011,"t":47,"m":9,"f":38,"mr":842,"fr":274,"mp":0.00016379122079056562,"fp":0.0007968962986264024},{"y":2012,"t":40,"m":5,"f":35,"mr":1266,"fr":289,"mp":0.0000926389120486169,"fp":0.0007402081042213011},{"y":2013,"t":59,"m":13,"f":46,"mr":646,"fr":216,"mp":0.00024232482711055605,"fp":0.0009888432683419679},{"y":2014,"t":80,"m":15,"f":65,"mr":584,"fr":164,"mp":0.0002729158327571777,"fp":0.00134958370533397},{"y":2015,"t":67,"m":9,"f":58,"mr":859,"fr":187,"mp":0.0001639433848844199,"fp":0.0011927775264261918},{"y":2016,"t":90,"m":11,"f":79,"mr":747,"fr":128,"mp":0.0002048112013107917,"fp":0.0016464164391555344},{"y":2017,"t":77,"m":9,"f":68,"mr":882,"fr":152,"mp":0.0001678666020069385,"fp":0.001445086705202312},{"y":2018,"t":91,"m":9,"f":82,"mr":885,"fr":113,"mp":0.00017270494319926314,"fp":0.001775237600398346},{"y":2019,"t":93,"m":11,"f":82,"mr":754,"fr":120,"mp":0.00021222001427661914,"fp":0.0017721300138312586},{"y":2020,"t":61,"m":7,"f":54,"mr":1064,"fr":188,"mp":0.00013966759113310322,"fp":0.0012176971992964416},{"y":2021,"t":68,"m":14,"f":54,"mr":644,"fr":191,"mp":0.000278834471907427,"fp":0.0011964372756680108},{"y":2022,"t":103,"m":13,"f":90,"mr":734,"fr":100,"mp":0.0002555784920868967,"fp":0.001979762428508579}]