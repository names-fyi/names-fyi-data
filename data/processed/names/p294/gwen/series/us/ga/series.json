[{"y":1929,"t":5,"m":0,"f":5,"mr":null,"fr":496,"mp":0,"fp":0.00016640042598509052},{"y":1930,"t":5,"m":0,"f":5,"mr":null,"fr":505,"mp":0,"fp":0.0001648750247312537},{"y":1932,"t":7,"m":0,"f":7,"mr":null,"fr":409,"mp":0,"fp":0.0002277533756303888},{"y":1934,"t":5,"m":0,"f":5,"mr":null,"fr":493,"mp":0,"fp":0.0001650491846570278},{"y":1936,"t":13,"m":0,"f":13,"mr":null,"fr":303,"mp":0,"fp":0.00044261346225869054},{"y":1938,"t":8,"m":0,"f":8,"mr":null,"fr":389,"mp":0,"fp":0.0002596812412763333},{"y":1939,"t":10,"m":0,"f":10,"mr":null,"fr":353,"mp":0,"fp":0.00032139872726104005},{"y":1940,"t":14,"m":0,"f":14,"mr":null,"fr":295,"mp":0,"fp":0.00044090322174282747},{"y":1941,"t":5,"m":0,"f":5,"mr":null,"fr":518,"mp":0,"fp":0.0001536523155403952},{"y":1942,"t":11,"m":0,"f":11,"mr":null,"fr":359,"mp":0,"fp":0.00031270432384796886},{"y":1943,"t":15,"m":0,"f":15,"mr":null,"fr":309,"mp":0,"fp":0.0004105539741624699},{"y":1944,"t":13,"m":0,"f":13,"mr":null,"fr":333,"mp":0,"fp":0.00036452345568235985},{"y":1945,"t":8,"m":0,"f":8,"mr":null,"fr":412,"mp":0,"fp":0.00023263252784320567},{"y":1946,"t":17,"m":0,"f":17,"mr":null,"fr":307,"mp":0,"fp":0.0004302054863852617},{"y":1947,"t":10,"m":0,"f":10,"mr":null,"fr":411,"mp":0,"fp":0.00022891676586393189},{"y":1948,"t":11,"m":0,"f":11,"mr":null,"fr":388,"mp":0,"fp":0.0002611771968563763},{"y":1949,"t":10,"m":0,"f":10,"mr":null,"fr":406,"mp":0,"fp":0.0002347142354183781},{"y":1950,"t":12,"m":0,"f":12,"mr":null,"fr":369,"mp":0,"fp":0.0002867314998446871},{"y":1951,"t":15,"m":0,"f":15,"mr":null,"fr":335,"mp":0,"fp":0.0003495362818660577},{"y":1952,"t":8,"m":0,"f":8,"mr":null,"fr":475,"mp":0,"fp":0.00018162005085361425},{"y":1953,"t":9,"m":0,"f":9,"mr":null,"fr":463,"mp":0,"fp":0.00020194314178652366},{"y":1954,"t":15,"m":0,"f":15,"mr":null,"fr":357,"mp":0,"fp":0.00032652705820889025},{"y":1955,"t":17,"m":0,"f":17,"mr":null,"fr":339,"mp":0,"fp":0.0003692601763760372},{"y":1956,"t":14,"m":0,"f":14,"mr":null,"fr":386,"mp":0,"fp":0.0002996831920541142},{"y":1957,"t":16,"m":0,"f":16,"mr":null,"fr":358,"mp":0,"fp":0.00034781096474066343},{"y":1958,"t":28,"m":0,"f":28,"mr":null,"fr":257,"mp":0,"fp":0.0006205811299009287},{"y":1959,"t":30,"m":0,"f":30,"mr":null,"fr":243,"mp":0,"fp":0.0006702712364270075},{"y":1960,"t":17,"m":0,"f":17,"mr":null,"fr":351,"mp":0,"fp":0.0003816793893129771},{"y":1961,"t":20,"m":0,"f":20,"mr":null,"fr":333,"mp":0,"fp":0.0004490547397727783},{"y":1962,"t":29,"m":0,"f":29,"mr":null,"fr":255,"mp":0,"fp":0.0006640866518582976},{"y":1963,"t":17,"m":0,"f":17,"mr":null,"fr":356,"mp":0,"fp":0.00038976522377109316},{"y":1964,"t":18,"m":0,"f":18,"mr":null,"fr":350,"mp":0,"fp":0.0004102470598960707},{"y":1965,"t":14,"m":0,"f":14,"mr":null,"fr":384,"mp":0,"fp":0.000342775995886688},{"y":1966,"t":17,"m":0,"f":17,"mr":null,"fr":338,"mp":0,"fp":0.00043946953442080503},{"y":1967,"t":6,"m":0,"f":6,"mr":null,"fr":623,"mp":0,"fp":0.00016268980477223427},{"y":1968,"t":13,"m":0,"f":13,"mr":null,"fr":394,"mp":0,"fp":0.0003452855245683931},{"y":1969,"t":7,"m":0,"f":7,"mr":null,"fr":606,"mp":0,"fp":0.0001833228577414624},{"y":1972,"t":8,"m":0,"f":8,"mr":null,"fr":544,"mp":0,"fp":0.0002223272100714226},{"y":1973,"t":5,"m":0,"f":5,"mr":null,"fr":749,"mp":0,"fp":0.0001467480629255694},{"y":1974,"t":5,"m":0,"f":5,"mr":null,"fr":749,"mp":0,"fp":0.00015151056028605195},{"y":2017,"t":5,"m":0,"f":5,"mr":null,"fr":1564,"mp":0,"fp":0.00010625637538252295},{"y":2020,"t":6,"m":0,"f":6,"mr":null,"fr":1348,"mp":0,"fp":0.00013529968881071573},{"y":2021,"t":8,"m":0,"f":8,"mr":null,"fr":1114,"mp":0,"fp":0.00017724996676563124}]