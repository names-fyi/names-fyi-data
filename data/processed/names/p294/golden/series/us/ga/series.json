[{"y":1914,"t":5,"m":5,"f":0,"mr":431,"fr":null,"mp":0.00023806122934818836,"fp":0},{"y":1915,"t":8,"m":8,"f":0,"mr":348,"fr":null,"mp":0.0003246357992127582,"fp":0},{"y":1916,"t":5,"m":5,"f":0,"mr":481,"fr":null,"mp":0.00019142419601837673,"fp":0},{"y":1917,"t":8,"m":8,"f":0,"mr":356,"fr":null,"mp":0.0003055534336567107,"fp":0},{"y":1919,"t":8,"m":8,"f":0,"mr":402,"fr":null,"mp":0.00025793139025019347,"fp":0},{"y":1922,"t":6,"m":0,"f":6,"mr":null,"fr":521,"mp":0,"fp":0.00016056948644525918},{"y":1937,"t":5,"m":5,"f":0,"mr":466,"fr":null,"mp":0.00017179178835251675,"fp":0}]